src/displayapp/screens/Notifications.cpp

```diff
@@ -285,6 +364,19 @@ void Notifications::NotificationItem::OnCallButtonEvent(lv_obj_t* obj, lv_event_
   running = false;
 }
 
+void Notifications::AnimateDismiss() {
+  dismissAnimTickCount = xTaskGetTickCount();
+}
+
+bool Notifications::AnimationElapsed() const {
+  bool elapsed = dismissAnimTickCount != 0 && xTaskGetTickCount() > dismissAnimTickCount + dismissAnimLength;
+  return elapsed;
+}
+
+void Notifications::AnimationReset() {
+  dismissAnimTickCount = 0;
+}
+
```

Comment:
I don't think comparing ticks are necessary, as the sideways transition is blocking. I replaced this with just a boolean for should show a new notification and it seemed to work fine.

Reply:
Yep, indeed, it seems like `dismissAnimTickCount` could be a simple boolean, as the whole animation will be handled in the lv task at once.

Reply:
renamed variable and made it to be a boolean